x/auth/keeper/migrations.go

```diff
@@ -45,6 +47,36 @@ func (m Migrator) Migrate4To5(ctx context.Context) error {
 	return v5.Migrate(ctx, m.keeper.KVStoreService, m.keeper.AccountNumber)
 }
 
+// Migrate5To6 migrates the x/auth module state from the consensus version 5 to 6.
+// It migrates the GlobalAccountNumber from x/auth to x/accounts .
+func (m Migrator) Migrate5To6(ctx context.Context) error {
+	return v6.Migrate(ctx, m.v56SetAccounts)
+}
+
+func (m Migrator) v56SetAccounts(ctx context.Context) error {
```

Comment:
this migration is too long, i think the best is just to do Accounts.AccountNumber.Set(auth.AccountNumber)

Reply:
i figure we can just update Accounts.AccountNumber in the upgrade handler